kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/matchers.kt

```diff
@@ -14,11 +14,22 @@ import io.kotest.matchers.MatcherResult
  * regardless of order.
  *
  */
-fun equalJson(expected: JsonTree, mode: CompareMode, order: CompareOrder) = object : Matcher<JsonTree> {
+fun equalJson(
+   expected: JsonTree,
+   mode: CompareMode,
+   order: CompareOrder,
+   ignoreUnknownKeys: Boolean = false
+) = object : Matcher<JsonTree> {
```

Comment:
I suppose changing this breaks binary compatibility. Is that allowed for a minor release?

Reply:
No, but this woud go into 5.0 so I'm ok with minor breaking changes (please add to changelog).